Chaincase/Views/ReceivePage.xaml

```diff
@@ -12,11 +12,12 @@
             <!-- ADDRESSES -->
             <StackLayout>
 
-            <Entry Text="{Binding Label}"
+            <Entry x:Name="LabelField"
```

Comment:
I see you're renaming this "LabelField" to differentiate the view from the viewmodel in the codebehind. I like to shorthand `v` for view and `vm` there so I can differentiate these